### ISSUE TITLE
Object detection with webcam using OpenCV

### ISSUE THREAD

**Issue by balavenkatesh3322** [tensorflow/models#5339]
I have done below feature in object detection.
1) We can identify an object via webcam(internal & external) using OpenCV tool.
2) Detect the object using OpenCV without installing Tensorflow using Docker compose file. Just download 
    the tensorflow repository and run the command(docker-compose run).

****Note**: I have completed that. If you ok, I will send a pull request.**

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by balavenkatesh3322** [comment]
@tensorflowbutler  N/A

**Comment by qlzh727** [comment]
It is not clear to me what is the feature request here. Are you trying to add a new model or update an existing model? The blog post does not show any code snippet, and seems only talking about how to config the environment. 

If you have a PR, feel free to send it and we can have the discussion there.

**Comment by balavenkatesh3322** [comment]
@qlzh727, Here I am not speaking about model. Once we download tensorflow model we can easily setup environment using docker and test the model using openCV.

Issues:- We need to setup environment before going to tensorflow model it will take more time and complications.

**Comment by balavenkatesh3322** [comment]
Yep. As well I will send a docker file for setup environment and python file for openCV tool configuration.

Developers or user easily test the Model.

**Comment by qlzh727** [comment]
Assigning to someone on object detection team.
